plugins/tmux/tmux.plugin.zsh

```diff
@@ -76,7 +80,11 @@ function _zsh_tmux_plugin_run() {
     elif [[ -e "$ZSH_TMUX_CONFIG" ]]; then
       tmux_cmd+=(-f "$ZSH_TMUX_CONFIG")
     fi
-    $tmux_cmd new-session
+    if [[ "$ZSH_TMUX_SESSIONGROUP" == "true" ]]; then
```

Comment:
Looks good, but the same conditional needs to happen when we call `tmux attach` on L74.

Reply:
It seems `attach-session` doesn't allow specifying a session group name.
```
attach-session [-dErx] [-c working-directory] [-t target-session]
```